lib/docs/stories/introduction/usage.stories.mdx

```diff
@@ -1,42 +1,63 @@
 import { Meta } from "@storybook/addon-docs/blocks";
 
-<Meta title="Introduction/Usage" />
+<Meta title="Introduction/Getting Started" />
 
-# Usage
+# Getting Started
```

Comment:
We should probably mimic all these changes in README.md which is shown on GitHub as well as on npmjs.org